src/Microsoft.HttpRepl.Tests/OpenApi/OpenApiV3EndpointMetadataReaderTests.cs

```diff
@@ -77,21 +77,23 @@ public void ReadMetadata_WithNoRequestBody_ReturnsEndpointMetadataWithEmptyAvail
         }
 
         [Fact]
-        public void ReadMetadata_WithNoContent_ReturnsEndpointMetadataWithEmptyAvailableRequests()
+        public void ReadMetadata_WithNoContent_ReturnsEndpointMetadataWithRequestButNoContentTypes()
```

Comment:
Is this a valid test case?
ReadMetadata_WithContent_ReturnsEndpointMetadataWithRequestAndContentTypes

Reply:
I added a couple tests for WithContent - when there's one and when there's more than one - in 9f1d12d.